sphinx/directives/patches.py

```diff
@@ -130,8 +130,6 @@ def run(self):
                                 nowrap='nowrap' in self.options)
         ret = [node]
         set_source_info(self, node)
-        if hasattr(self, 'src'):
-            node.source = self.src
```

Comment:
🎉 

Memo:

This is for docutils 0.8 or older. Directive.src instance attribute has been deleted at: docutils-mirror/docutils@5d61d6b#diff-6d19a4ff14e843a848631efc1195411f